ansible_builder/cli.py

```diff
@@ -80,6 +80,19 @@ def add_container_options(parser):
         )
     )
 
+    # Keyring option copied over from ansible-galaxy. Gets passed to the invocations of it verbatim.
+    create_command_parser.add_argument(
+        '--keyring',
+        help='The keyring used during signature verification when installing collections from a Galaxy server')
+    # Disable-gpg-verify option copied over from ansible-galaxy. Gets passed to the invocations of it verbatim.
+    create_command_parser.add_argument(
+        '--disable-gpg-verify',
+        dest='disable_gpg_verify',
+        action='store_true',
```

Comment:
With a true pass-through of this option, I think this might be a bit confusing as it is with a default of `True`. Looking at the help output, a user would assume that they must use this option in order to NOT turn on collection verification, when the opposite is true. Actually, as-is, there is no way to turn on verification since there is no way to set this value to `False`.

If we want the user to specifically opt-in to verification, I think we'd have to go with an option like `--enable-collection-verfication` (bikeshed the name) which defaults to `False`. If that option is not given, we use `--disable-gpg-verify` during collection installation. If it is given, then we pass the keyring value along. We also probably want to verify that a keyring value is actually given if we enable signature verification.

Reply:
Resolved this by turning on sig verify if `--keyring` is given. No need for the other option.